Apps/Admin/Tests/Functional/cypress/integration/e2e/delegation.cy.js

```diff
@@ -93,3 +94,131 @@ describe("Delegation", () => {
         cy.get("[data-testid=delegate-table]").should("not.exist");
     });
 });
+
+describe("Delegation Protect", () => {
+    beforeEach(() => {
+        cy.login(
+            Cypress.env("keycloak_username"),
+            Cypress.env("keycloak_password"),
+            "/delegation"
+        );
+    });
+
+    it("Verify protect dependent toggle, save button, cancel confirmation button and cancel button are selected.", () => {
+        cy.get("[data-testid=query-input]")
+            .clear()
+            .type(dependentWithGuardian.phn);
+        cy.get("[data-testid=search-button]").click();
+
+        // Protect dependent toggle
+        cy.get("[data-testid=dependent-protected-switch]").should(
+            "have.attr",
+            "aria-checked",
+            "false"
+        );
+
+        // Protect
+        cy.get("[data-testid=dependent-protected-switch]").click();
+        cy.get("[data-testid=dependent-protected-switch]").should(
+            "have.attr",
+            "aria-checked",
+            "true"
+        );
+        cy.get("[data-testid=cancel-edit-button]").should(
+            "be.visible",
+            "be.enabled"
+        );
+
+        // Save button
+        cy.get("[data-testid=save-button]").click();
+
+        // Confirmation dialog confirmation button
+        cy.get("[data-testid=confirm-button]").should(
+            "be.visible",
+            "be.enabled"
+        );
+
+        // Cancel confirmation
+        cy.get("[data-testid=cancel-button]").click();
+
+        cy.get("[data-testid=dependent-protected-switch]").should(
+            "have.attr",
+            "aria-checked",
+            "true"
+        );
+
+        // Save button
+        cy.get("[data-testid=save-button]").should("be.visible", "be.enabled");
+
+        // Cancel button
+        cy.get("[data-testid=cancel-edit-button]").click();
+
+        cy.get("[data-testid=dependent-protected-switch]").should(
+            "have.attr",
+            "aria-checked",
+            "false"
+        );
+    });
+
+    it("Verify protect/unprotect dependent toggle, select to remove, save button and confirmation button are selected.", () => {
+        cy.get("[data-testid=query-input]").clear().type(protectDependent);
+        cy.get("[data-testid=search-button]").click();
+
+        // Confirm delegate table
+        getTableRows("[data-testid=delegate-table]").should("have.length", 2);
```

Comment:
Testing this makes the test brittle, without value.

For the dependent timeline test on another ticket I'm going to be adding a new dependent to the seed file, thus testing for 2 will break but it also doesn't add any value. I would argue that extracting the number of rows x (testing should check that its > 0) and then the result of removing the dependent should be equal to x - 1 will provide your result without being brittle.